Source/CLI/Main.cpp

```diff
@@ -65,7 +69,7 @@ void WriteToDisk(uint64_t ID, raw_frame* RawFrame, void* Opaque)
 }
 
 //---------------------------------------------------------------------------
-void WriteToDisk(uint8_t* Buffer, size_t Buffer_Size, void* Opaque)
+void WriteToDisk(uint8_t* Buffer, uint64_t Buffer_Size, void* Opaque)

```

Comment:
should stay size_t
-> need to replace the signature instead:
typedef void(*write_file_call)(uint8_t*, uint64_t, void*);
is wrong.